lib/project/ext/ui_view.rb

```diff
@@ -82,7 +82,12 @@ def image
   def stylesheet
     rmq.stylesheet
   end
+
   def stylesheet=(value)
     rmq.stylesheet = value
   end
+
+  def find(*args)
+    rmq(self).find(args)
```

Comment:
I mirrored this after what we had in object, but yeah, I think you're right - I'll throw together a test and verify and get another PR together.  Thanks @jamonholmgren